src/KafkaFlow/Consumers/PartitionOffsets.cs

```diff
@@ -7,42 +7,42 @@ namespace KafkaFlow.Consumers
 
     internal class PartitionOffsets
     {
-        private readonly SortedSet<long> pendingOffsets = new();
-        private readonly LinkedList<long> offsetsOrder = new();
+        private readonly SortedSet<long> processedOffsets = new();
+        private readonly LinkedList<long> receivedOffsets = new();
 
-        public long LastOffset { get; private set; } = -1;
+        public long LastCommittedOffset { get; private set; } = -1;
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void AddOffset(long offset)
+        public void Store(long offset)
```

Comment:
I think Store() can lead to confusion here because "store" is the term used to store an offset to be committed